lib/dogapi/common.rb

```diff
@@ -170,4 +169,10 @@ def Dogapi.find_localhost
       raise 'Cannot determine local hostname via hostname -f'
     end
   end
+
+  def Dogapi.find_proxy
+    ENV['DOGAPI_HTTPS_PROXY'] || ENV['dogapi_https_proxy'] ||
```

Comment:
Looking at our documentation re: proxy env vars: https://docs.datadoghq.com/agent/proxy/?tab=agentv6v7#environment-variables, our Datadog agent uses `DD_PROXY_HTTPS` and `DD_PROXY_HTTP` and they take precedence over `HTTPS_PROXY` and `HTTP_PROXY`. Would you not want to use the former 2 instead of `DOGAPI_HTTP(S)_PROXY` (for consistency - even if this client library does not use the agent)?

Reply:
I've changed `DOGAPI_HTTP(S)_PROXY` to `DD_PROXY_HTTP(S)` and did the same for the  lower cased variable.